extensions/amp-iframe/0.1/amp-iframe.js

```diff
@@ -466,6 +466,8 @@ export class AmpIframe extends AMP.BaseElement {
       /*opt_allowOpaqueOrigin*/ true
     );
 
+    addEventListener('message', this.listenForPymMessage_.bind(this));
```

Comment:
This seems not ideal. There should rather be one single `message` event handler that all of the iframes share. 

It seems rather that `listenFor` should be used, but the problem is that the Pym.js `event.data` is a string and not an object with any `sentinel`. So there isn't an easy way to integrate with this existing event listener system.